styles/styles.css

```
@@ -30,100 +30,231 @@ body {
   background-color: #ffffff;
 }
 
-a {
-  text-decoration: none;
+.page-wrapper {
+  width: 1200px;
+  margin: 0 auto;
+}
+
+.title {
+  text-transform: uppercase;
+}
+
+.button {
+  text-transform: uppercase;
+}
+
+.list {
+  list-style: none;
 }
```

давай тут еще добавим
```suggestion
.list {
  margin: 0;
  padding: 0;
  list-style: none;
}
```

и добавим этот класс всем спискам вообще